CONTRIBUTING.md

```diff
@@ -29,6 +30,37 @@ First of all, thanks for taking the time to contribute! This project can only gr
   <li>"icon" (lowercase) refers specficially to the font icon versions of the Icons.</li>
 </ol>
 
+<hr>
+<h2 id='iconRequirements'>What Icons Do We Accept?</h2>
+<p>Devicon only accepts Icons of development languages and tools. </p>
+<p><b>Development</b> refers to programming or programming related jobs.</p>
+<p><b>Tools</b> can be software, OS, services, etc... that helps with development. It must be specifically related to development (so softwares like Microsoft Word or Google Calendar won't be accepted since it's too general).</p>
+
+<b>Special Cases (see <a href='https://github.com/devicons/devicon/discussions/353'>this discussion</a> for more details)</b>
+<ul>
+  <li>Tech companies used to be accepted in the repo. However, we do not accept them any more. Icons like Facebook, Twitter, etc... are kept due to backwards compatibility. </li>
+    <ul>
+    <li>We still accept their icons if they represents a service and not the company itself.</li>
+    <li>Ex. AWS is accepted since their names refer to their services. We will not accept Amazon since that's the parent company that includes non-tech related tools</li>
+    </ul>
+  <li>Related fields like graphic designs or game development. Since many "development" jobs require people to know related fields, some softwares will be accepted in Devicon even though they aren't strictly "development tools". </li>
+    <ul>
+      <li>ex. some Adobe products, game engines, CMS, etc... See <a href='https://github.com/devicons/devicon/pull/336'> for more details.</a></li>
+    </ul>
+  <li>General tools that are well known in the tech industries. Things like Trello, Slack, etc... are accepted under this category. Unfortunately, we won't accept Discord since its use are still too general. This will be treated on a case by case basis.</li>
```

Comment:
```suggestion
  <li>General tools that are well known in the tech industries. Things like Trello, Slack, etc., are accepted under this category. Unfortunately, we won't accept Discord since its use is still too general. This will be treated on a case-by-case basis.</li>
```